src/main/java/uk/gov/companieshouse/officer/delta/processor/tranformer/TransformerUtils.java

```diff
@@ -27,8 +27,8 @@ private TransformerUtils() {
         // utility class; prevent instantiation
     }
 
-    public static boolean parseYesOrNo(final String serviceAddressSameAsRegisteredAddress) {
-        return "Y".equalsIgnoreCase(serviceAddressSameAsRegisteredAddress);
+    public static boolean parseYesOrNo(final String fieldValue) {
+        return "Y".equalsIgnoreCase(fieldValue);
```

Comment:
Would `BooleanUtils.toBoolean(String)` be useful instead here?

Reply:
It may be, but would I be right to be wary of then affecting the usages of this in serviceAddressSameAsRO and the Pre92Appointment code?